### DISCUSSION TITLE
Can we use gptel with Code Llama 70B?

### DISCUSSION THREAD

**Discussion by bbigras** [karthink/gptel#198] [Q&A] [answered]
Code Llama 70B just released and I have no idea how to use it :sweat: .

I saw https://github.com/karthink/gptel/issues/121 but I have no idea if we could use code llama with it.

**Comment by Zambito1** [comment, marked as answer]
https://ollama.ai/library/codellama/tags

70B was last updated yesterday. gptel can be used with ollama, so looks like you should be able to do that :)

**Reply by bbigras** [reply]
Thanks, yeah this should work:

```lisp
(gptel-make-ollama "Ollama"
  :stream t
  :models '("codellama:70b"))
```

well not on my laptop but I'll try on my desktop tomorrow.

**Reply by karthink** [reply]
It should also work via `llama.cpp`.
